doc/component/form-textarea.md

```diff
@@ -1,2 +1,37 @@
 # form-textarea
-[TBD]
+The `<form-textarea />` component allows you to render a textarea block.
+
+## Attributes
+
+The following attributes (bindables) are available for this component.
+
+| attribute | type | default | description |
+|---|---|---|---|
+| name | string | `''` | The name of the textarea |
+| classes | string | `''` | Classes to add |
+| value | string | `''` | The text to display within the textarea |
+| placeholder | string | `''` | A short hint that describes the expected value |
+| autofocus | boolean | `false` | Specifies to autofocus the textarea |
+| required | boolean | `false` | Make the textarea required |
+| cols | number | | The visible width of the textarea |
+| rows | number | | The visible number of lines in the textarea |
+| disabled | boolean | `false` | Specifies that the textarea should be disabled |
+| maxlength | number | | The maximum number of characters allowed in the text area |
+| readonly | boolean | `false` | Specifies that the textarea should be read-only |
+| wrap | string | `'hard'` | Specifies how the text in a text area is to be wrapped when submitted in a form |
+
+## Example
+```html
+<aurelia-textarea 
+    name.bind="message" 
+    classes.bind="form-control" 
+    value="Hello!" 
+    placeholder="A message" 
+    autofocus.bind="true" 
+    required.bind="true" 
+    disabled.bind="false" 
+    maxlength.bind="500" 
+    readonly.bind="false" 
+    wrap.bind="hard">
```

Comment:
You can't bind a string. Use regular attribute or supply a view model in the example. (I recommend using regular attributes)